iconless campaigns display 'no icon' text instead of blank; closes #1082

Dylan-RG-Patterson opened the pull request:
campaigns with no icons appearing in list or detail views will now display "none" or "no icon" text respectively.
![image](https://user-images.githubusercontent.com/105619909/179337468-f6ecf5bc-d132-43de-a854-07ed673968cf.png)

![image](https://user-images.githubusercontent.com/105619909/179337456-add13301-3e1c-412c-8e3a-5b508a466c57.png)

tylerecouture reviewed (changes requested):
Hmmm... Doesn't get_icon_url() return the default icon?  I think the problem was that the list view was using some extra uneccessary logic instead of just using get_icon_url which will always return an icon.  I didn't notice that.

So.... return the detail view how it was, which should have been fine, and just use get_icon_url method in the list view.

Dylan-RG-Patterson commented:
![image](https://user-images.githubusercontent.com/105619909/179638078-d0ba576f-8387-414e-a9f5-23390c2eab72.png)
changed list view to now display default icon, and reverted change to detail view.
